packages/react-preferences/src/components/UnsubscribePage.tsx

```diff
@@ -0,0 +1,260 @@
+                  fill="#2C1437"
+                />
+                <path
+                  d="M25.1 28.7C26.3702 28.7 27.4 27.6703 27.4 26.4C27.4 25.1298 26.3702 24.1 25.1 24.1C23.8297 24.1 22.8 25.1298 22.8 26.4C22.8 27.6703 23.8297 28.7 25.1 28.7Z"
+                  fill="#2C1437"
+                />
+                <path
+                  d="M25.6 40.8C26.2 44.4 29 55.9 41.8 58C42 58 42.1 57.8 41.9 57.7C39 56.3 31.2 51.9 26 40.7C25.8 40.6 25.6 40.6 25.6 40.8Z"
+                  fill="#2C1437"
+                />
+                <path
+                  d="M141.2 54.9C141.2 71.1 128.9 83.2 112.8 83.2C96.7 83.2 84.5 71.1 84.5 54.9C84.5 38.5 96.7 26.3 112.8 26.3C128.9 26.3 141.2 38.5 141.2 54.9ZM102 54.9C102 61.3 106.7 66.1001 112.9 66.1001C119.2 66.1001 123.9 61.3 123.9 54.9C123.9 48.2 119.2 43.4 112.9 43.4C106.6 43.4 102 48.3 102 54.9Z"
+                  fill="#2C1437"
+                />
+                <path
+                  d="M199.8 56.7C199.8 72.2 188.9 83.2 173.6 83.2C158.2 83.2 147.2 72.2 147.2 56.7V27.4H165V56.7C165 62.5 168.3 66.1 173.7 66.1C178.8 66.1 182.2 62.4 182.2 56.7V27.4H199.8V56.7Z"
+                  fill="#2C1437"
+                />
+                <path
+                  d="M243.6 27V43.2C241.6 42.9 239.7 42.5 237.9 42.5C230.5 42.5 224.5 46.4 224.5 56.8V82.1H206.8V27.4H224.5V33.2C227.7 28.9 232.4 26.4 238.4 26.4C240 26.4 241.7 26.6 243.6 27Z"
+                  fill="#2C1437"
+                />
+                <path
+                  d="M269.9 10.8C269.9 16.6 265.1 21.4 259.4 21.4C253.6 21.4 248.8 16.5 248.8 10.8C248.8 5.10003 253.7 0.300027 259.4 0.300027C265.1 0.200027 269.9 5.10003 269.9 10.8ZM268.1 27.4V82.1H250.4V27.4H268.1Z"
+                  fill="#2C1437"
+                />
+                <path
+                  d="M329.4 59.4H290.4C291.7 65.1 296.4 68.6 302.5 68.6C307.8 68.6 310.8 66.4 311.6 63.4H329C327.3 75.5 317 83.3 303 83.3C286.8 83.3 274.2 70.8 274.2 54.7C274.2 38.7 286.6 26.4 302.7 26.4C317.6 26.4 329.9 38.2 329.9 53.1C329.8 54.5 329.6 57.4 329.4 59.4ZM311.2 48.2C310.8 43.8 306.5 40.8 301.8 40.8C296.9 40.8 293 43 291.2 48.2H311.2Z"
+                  fill="#2C1437"
+                />
+                <path
+                  d="M372.7 27V43.2C370.7 42.9 368.8 42.5 367 42.5C359.6 42.5 353.6 46.4 353.6 56.8V82.1H335.9V27.4H353.6V33.2C356.8 28.9 361.5 26.4 367.5 26.4C369.1 26.4 370.8 26.6 372.7 27Z"
+                  fill="#2C1437"
+                />
+              </svg>
+            )}
+          </ImageWrapper>
+          <HeaderText>Notification Center</HeaderText>
+        </div>
+      </Header>
+      <Body>
+        <div>
+          <p>{getText(topic.defaultStatus === "REQUIRED")}</p>
+          <TopicWrapper>
+            <div>
+              <h1>{topic.templateName}</h1>
+              <p>Be notified if you are invited to a group or event.</p>
```

Comment:
Do we want this as a static description? What if notifications are unrelated to invites?

Reply:
We're going to add topic descriptions eventually, but we don't have them yet.